### PR TITLE
FMD-232: Add script to extract download logs into reporting output

### PR DESCRIPTION
**Pull request by gidsg**
### Change description
Adds script to generate a report based on logging added in #88 

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
Can be ran by users with AWS access

### CODE REVIEW

File: scripts/extract_download_logs.py
Comment (cyrusdobbs) [inline scripts/extract_download_logs.py:1]: Module-level docstring with args, output, description and set up would be useful. Could include a link to the aws-vault confluence docs as I presume that's what you need to have (or similar?) as a prerequisite to running this?

**Review by cyrusdobbs — COMMENTED** [review]
Nice 👍

File: scripts/extract_download_logs.py
Comment (gidsg) [reply scripts/extract_download_logs.py:1]: @cyrusdobbs OK I added a module docstring in 5a3f0fe
I kept it quite high level as I didn't want to repeat the information already outputted by the options parser.

**Review by cyrusdobbs — APPROVED** [review]
LGTM

**Pull request merged**